src/dashboard/src/installer/steps.py

```diff
@@ -0,0 +1,145 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Archivematica.  If not, see <http://www.gnu.org/licenses/>.
+
+import json
+import logging
+import socket
+import urlparse
+import uuid
+
+from django.conf import settings as django_settings
+from django.contrib.auth import get_user_model
+import requests
+from tastypie.models import ApiKey
+
+from main.models import Agent, User
+import components.helpers as helpers
+import storageService as storage_service
+import utilities.FPRClient.client as FPRClient
+from version import get_version
+
+
+logger = logging.getLogger('archivematica.dashboard')
+
+
+def create_super_user(username, email, password, key):
+    UserModel = get_user_model()
+    user = UserModel._default_manager.db_manager('default').create_superuser(username, email, password)
+    api_key = ApiKey.objects.create(user=user)
+    api_key.key = key
+    api_key.save()
+
+
+def setup_pipeline(org_name, org_identifier):
+    # Assign UUID to Dashboard
+    dashboard_uuid = str(uuid.uuid4())
+    helpers.set_setting('dashboard_uuid', dashboard_uuid)
+
+    # Update Archivematica version in DB
+    archivematica_agent = Agent.objects.get(pk=1)
+    archivematica_agent.identifiervalue = "Archivematica-" + get_version()
+    archivematica_agent.save()
+
+    if org_name != '' or org_identifier != '':
+        agent = get_agent()
+        agent.name = org_name
+        agent.identifiertype = 'repository code'
+        agent.identifiervalue = org_identifier
+        agent.save()
+
+
+def submit_fpr_agent():
+    agent = get_agent()
+    url = django_settings.FPR_URL + 'agent/'
+    resp = {}
+    headers = {'Content-Type': 'application/json'}
+    payload = {
+        'uuid': helpers.get_setting('dashboard_uuid'),
+        'agentType': 'new install',
+        'agentName': agent.name,
+        'clientIP': get_my_ip(),
+        'agentIdentifierType': agent.identifiertype,
+        'agentIdentifierValue': agent.identifiervalue
+    }
+
+    try:
+        logger.info("FPR Server URL: {}".format(django_settings.FPR_URL))
+        r = requests.post(url, data=json.dumps(payload), headers=headers, timeout=120, verify=True)
+        if r.status_code == 201:
+            resp['result'] = 'success'
+        else:
+            resp['result'] = 'failed to fetch from ' + url
+    except:
+        resp['result'] = 'failed to post to ' + url
+
+    return resp
+
+
+def download_fpr_rules():
+    resp = {}
+    logger.info("FPR Server URL: {}".format(django_settings.FPR_URL))
+    fpr = FPRClient.FPRClient(django_settings.FPR_URL)
+    resp['result'], resp['response'], error = fpr.getUpdates()
+    if error:
+        logging.warning("FPR update error: {}".format(error))
+    return resp
+
+
+def setup_pipeline_in_ss(use_default_config=False):
```

Comment:
This seems like it's doing the opposite of what it was before. i.e. previously, the `use_default_config` setting meant `create_pipeline` would be called with args, and now it results in the call without args.